.github/workflows/ci.yml

```diff
@@ -2,34 +2,37 @@ name: CI
 
 on:
   push:
-  pull_request_target:
+  pull_request:
```

Comment:
Why this change?

Reply:
Because it wouldn't run my change on this PR, and I couldn't find any reference to `pull_request_target ` in the GitHub Actions documentation.

Reply:
Ok, that explains it then. I'll leave the change for now otherwise it's hard for me to test my changes. But I can change it back last minute.